.changeset/three-eels-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': minor
+---
+
+BREAKING CHANGE: The Image component will use the new `next/image` if the host is using Next.js v13.
```

Comment:
I think that there's an opportunity here to go into more detail as to what exactly users can expect to "break". At the very least a link to the Next.js documentation or blog post announcing the new Image. But ideally we're explaining what has changed, why, and what they can expect, etc.

Reply:
Yes, it's hard to write the complete release notes on different commits. This is just a draft, and we can go through the release notes together when we want to release the version.